wagtailspace/home/templates/home/home_page.html

```diff
@@ -1,15 +1,14 @@
 {% extends 'base.html' %}
-{% load wagtailcore_tags %}
-{% load render_bundle webpack_static from webpack_loader %}
```

Comment:
This probably makes the filename unique by adding a hash. Great for setting long-time cache headers, yet still 'invalidate' the browser cache by supplying a new filename.

You'd might consider something similar. Have a look at https://pypi.org/project/django-manifest-loader/